builders/Dockerfile.alpine

```diff
@@ -1,3 +1,14 @@
+# Install C compiler for cross-compilation. This is required by
+# Wasmer in https://github.com/wasmerio/wasmer/blob/2.2.1/lib/vm/build.rs.
+# For newer versions this might not be needed anymore since build.rs is removed
+# in https://github.com/wasmerio/wasmer/pull/2807.
+#
+# https://unix.stackexchange.com/questions/620205/aarch64-linux-musl-cross-has-a-broken-link-for-ld-musl-aarch64-so-1
+
+# We need aarch64-linux-musl-cross to compile, wget to musl.cc is denied by github actions.
+# We will use a copy from an image that already contains aarch64-linux-musl-cross.
+FROM mondoolabs/golang:1.15.0-alpine AS builder
```

Comment:
Please check the golang version.

Reply:
This image is only used to copy aarch64-linux-musl-cross. Therefore, the golang version of this image does not affect the build result.